corehq/apps/importer/templates/importer/excel_config.html

```diff
@@ -1,15 +1,10 @@
-{% extends "style/bootstrap2/two_column.html" %}
+{% extends "style/bootstrap3/two_column.html" %}
 {% load hq_shared_tags %}
 {% load report_tags %}
 {% load i18n %}
 
-{% block head %}
-    {{ block.super }}
-{% endblock %}
-
-{% block js %}
-    {{ block.super }}
-    <script src="{% static 'style/js/bootstrap2/main.js' %}"></script>
+{% block js %}{{ block.super }}
+    <script src="{% new_static 'style/js/bootstrap2/main.js' %}"></script>
```

Comment:
@biyeun merged with master this was `hqwebapp/js/main.js` which according to https://github.com/dimagi/commcare-hq/pull/10502/files translates to `bootstrap2/main.js` I'm assuming it should actually be moved to bootstrap3?